shared/js/dataDriver.js

```diff
@@ -85,15 +85,12 @@ loop.DataDriver = function() {
     }
 
     /**
-      * Add a page to the room's table of contents.
-      *
-      * @param {String} url       The url of the page.
-      * @param {String} title     The title of the page.
-      * @param {Object} metadata  Additional metadata,
-      *                           as provided by PageMetadata.
-      */
-    addPage(url, title, metadata) {
-      this.update("page", this.makeId(), { title, url, metadata });
+     * Add a page to the room's table of contents.
+     *
+     * @param {Object} page      The page object to add.
+     */
+    addPage(page) {
+      this.update("page", this.makeId(), { title: page.title, url: page.url });
```

Comment:
So you want it the same way I originally had it before you asked me to change it.